data/service/api/v1/summary.go

```diff
@@ -31,6 +37,8 @@ func SummaryRoutes() []dataService.Route {
 
 		dataService.Get("/v1/summaries/migratable/cgm", GetMigratableUserIDs[types.CGMStats, *types.CGMStats], api.RequireAuth),
 		dataService.Get("/v1/summaries/migratable/bgm", GetMigratableUserIDs[types.BGMStats, *types.BGMStats], api.RequireAuth),
+
+		dataService.Get("/v1/summaries/realtime/:clinicId", GetPatientsWithRealtimeData, api.RequireAuth),
```

Comment:
Is this desired schema? It would be mixing routes where /clinic* no longer goes specifically to the clinic service, I'm not sure whats better, I understand the intent though.

Reply:
I think so, yes. It better represents the relationship between the clinic and the generated report. [This](https://github.com/tidepool-org/development/blob/master/charts/tidepool/charts/hydrophone/templates/4-routetable.yaml#L165-L173) is an example where this pattern is already used.

Reply:
done